src/devices/bus/isa/sblaster_lle.cpp

```diff
@@ -191,29 +242,32 @@ void isa8_sblaster_2_0_lle_device::raise_irq()
 {
 	if(!m_irq_raised)
 	{
-		LOG("SB raising IRQ %d\n", 5);
-		switch(m_jp4->read())
+		LOG("SB raising IRQ %d\n", m_selected_irq);
+		switch(m_selected_irq)
 		{
-			case 1: m_isa->irq7_w(ASSERT_LINE); m_irq_raised = 7; break;
-			case 2: m_isa->irq5_w(ASSERT_LINE); m_irq_raised = 5; break;
-			case 4: m_isa->irq3_w(ASSERT_LINE); m_irq_raised = 3; break;
-			case 8: m_isa->irq2_w(ASSERT_LINE); m_irq_raised = 2; break;
+			case 7: m_isa->irq7_w(ASSERT_LINE); m_irq_raised = 7; break;
+			case 5: m_isa->irq5_w(ASSERT_LINE); m_irq_raised = 5; break;
+			case 3: m_isa->irq3_w(ASSERT_LINE); m_irq_raised = 3; break;
+			case 2: m_isa->irq2_w(ASSERT_LINE); m_irq_raised = 2; break;
 		}
```

Comment:
Wouldn’t it be better to `m_irq_raised = m_selected_irq` after the `switch` statement?  It would definitely be less clutter.